icu4c/source/i18n/chnsecal.cpp

```diff
@@ -325,8 +325,6 @@ const UFieldResolutionTable* ChineseCalendar::getFieldResolutionTable() const {
  * @stable ICU 2.8
  */
 int32_t ChineseCalendar::handleComputeMonthStart(int32_t eyear, int32_t month, UBool useMonth) const {
-    ChineseCalendar *nonConstThis = (ChineseCalendar*)this; // cast away const
```

Comment:
I am most happy to see one of these go away.
Twenty more to go.

Reply:
As far as I can see, this is the only one that tries to save, modify and then restore the internal state (please shout if you can find anywhere else where that is being done). While it would be nice to get rid of the rest too, I wouldn't be all too concerned.